pkg/xen-tools/Dockerfile.in

```diff
@@ -9,6 +9,7 @@ COPY initrd/base.files /etc/mkinitfs/features.d/base.files
 COPY initrd/init-initrd initrd/mount_disk.sh initrd/udhcpc_script.sh /
 COPY initrd/poweroff /sbin/poweroff
 COPY initrd/chroot2.c /tmp/
+COPY initrd/00000080 /etc/acpi/PWRF/
```

Comment:
Just curious -- why such a weird name for the script?

Reply:
It is the code of "button" from qemu system_powerdown command.

Reply:
@giggsoff makes perfect sense! thanks for the explanation